src/utils/request.ts

```diff
@@ -0,0 +1,16 @@
+import { parse as parseUrl } from 'url';
+import parseCurl from 'parse-curl';
+
+import { Request, Method } from '../domain/entity';
+
+export function createRequestFromCurl(curlCommand: string) {
+  const parsedCommand = parseCurl(curlCommand);
+  const url = parseUrl(parsedCommand.url);
+
+  return new Request(
+    parsedCommand.method as Method,
+    url.path || '',
+    parsedCommand.header,
+    parsedCommand.body || ''
```

Comment:
If the user edit the response `content-type` (to something like `application/json` for instance), then Memento will break until restarted (to run the migrations).

Reply:
An option: add a command `run migrations` for moving files to the appropriate file